Assets/MixedRealityToolkit/Inspectors/Utilities/MixedRealitySearchUtility.FieldSearchResult.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEditor;
```

Comment:
@julenka/@davidkline-ms and other MRTK people, not sure how the team feels about MRSearchUtility.FieldSearchResult as a file name. 

Feels like it should just be under the MRSearchUtility folder with everything else